src/riak_kv_status.erl

```diff
@@ -91,33 +91,60 @@ get_stats(console) ->
         ++ riak_kv_stat_bc:disk_stats()
         ++ riak_kv_stat_bc:app_stats().
 
-aliases() ->
-    Grouped = exometer_alias:prefix_foldl(
-                <<>>,
-                fun(Alias, Entry, DP, Acc) ->
-                        orddict:append(Entry, {DP, Alias}, Acc)
-                end, orddict:new()),
-    lists:keysort(
-      1,
-      lists:foldl(
-        fun({K, DPs}, Acc) ->
-                case exometer:get_value(K, [D || {D,_} <- DPs]) of
-                    {ok, Vs} when is_list(Vs) ->
-                        lists:foldr(fun({D,V}, Acc1) ->
-                                            {_,N} = lists:keyfind(D,1,DPs),
-                                            [{N,V}|Acc1]
-                                    end, Acc, Vs);
-                    Other ->
-                        Val = case Other of
-                                  {ok, disabled} -> undefined;
-                                  _ -> 0
-                              end,
-                        lists:foldr(fun({_,N}, Acc1) ->
-                                            [{N,Val}|Acc1]
-                                    end, Acc, DPs)
-                end
-        end, [], orddict:to_list(Grouped))).
 
+aliases() ->
+    AllStats =
+        exometer_alias:prefix_foldl(
+            <<>>,
+            fun(Alias, Entry, DP, Acc) -> [{Entry, {DP, Alias}}|Acc] end,
```

Comment:
Later on we make an assumption on how these entries are kind of sorted such that consecutive entries end up next to each other, right?
Would be good to provide this assumption in a comment. Otherwise it is weird that if 
`Entry == PrevEntry` you "randomly" discard the new entry.
A comment would explain this for future developers.

Reply:
Added comment.  Ordering is only required for efficiency.